astrocut/tests/test_cube_cut.py

```diff
@@ -150,61 +161,64 @@ def test_cutout_extras(tmpdir):
     cube_file = cube_maker.make_cube(ffi_files, path.join(tmpdir, "test_cube.fits"), verbose=False)
 
     # Making the cutout
-    myfactory = CutoutFactory()
     coord = "256.88 6.38"
 
     ###########################
     # Test  _parse_table_info #
```

Comment:
I recommend breaking up large tests into several smaller tests when possible. That way you can be more selective with @skip & @not_smoke, etc.